WildAidDemo/functions/searchFacetByBoardings/source.js

```diff
@@ -178,22 +175,25 @@ exports = function(limit, offset, query, filter, agenciesToShareData){
       }
     }
     
-    var amount = boardingsCollection.aggregate([agencyAggregate,
-      aggregateTerms, {
+    amount = boardingsCollection.aggregate([
+      aggregateTerms,
+      agencyAggregate, {
         '$count': "total"
       }
     ]).toArray();
 
-    var boardings = boardingsCollection.aggregate([agencyAggregate,
-      aggregateTerms, {
+    boardings = boardingsCollection.aggregate([
+      aggregateTerms,
+      agencyAggregate, {
         '$skip': offset
       }, {
         '$limit': limit
       }
     ]).toArray();
 
-    var highlighted = boardingsCollection.aggregate([agencyAggregate,
-      aggregateTerms, {
+    const highlighted = boardingsCollection.aggregate([
```

Comment:
does this need to be a `const`? @sebawita

Reply:
This would also work with `var`, however since `highlighted` won't be reassigned anywhere after this line, it is better to keep it as a const.
This way we can make sure that nobody introduces additional changes to this aggregation.